src/groups/mwc/mwcu/mwcu_sharedresource.t.cpp

```diff
@@ -232,7 +232,7 @@ static void test1_resource_creators()
         bslma::TestAllocator resourceFactory;
 
         // allocate a resource
-        int* resource = new (resourceFactory) int(42);
```

Comment:
Need to rollback changes in this file, because in the original sources it was allocating integer with a default value 42. It's not related to cast.

https://en.cppreference.com/w/cpp/language/new

Reply:
I see, this makes sense. I will rollback these changes